### PR TITLE
Trying to solve syslog problem on windows

### PR DESCRIPTION
**Pull request by vgarvardt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -204,6 +204,7 @@ Uses [`github.com/bshuster-repo/logrus-logstash-hook` implementation](https://gi
 
 #### `syslog`
 
+Not supported on Windows.
```
Comment (rafaeljesus) [inline README.md:207]: 😆

File: hooks.go [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+	"github.com/bshuster-repo/logrus-logstash-hook"
+	log "github.com/sirupsen/logrus"
+	"gopkg.in/gemnasium/logrus-graylog-hook.v2"
+)
+
+func (c LogConfig) initLogstashHook(h LogHook) error {
+	if err := c.validateRequiredHookSettings(h, []string{"network"}); err != nil {
+		return err
+	}
+	network, _ := h.Settings["network"]
```
Comment (rafaeljesus) [inline hooks.go:17]: I don't have the full picture of the lib, but I'd catch for `ok` here, but if ya think it isn't needed that's fine

File: hooks.go [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+	for k, v := range h.Settings {
+		if k != "host" && k != "port" && k != "async" {
+			extra[k] = v
+		}
+	}
+	var hook *graylog.GraylogHook
+	if async {
+		hook = graylog.NewGraylogHook(fmt.Sprintf("%s:%s", h.Settings["host"], h.Settings["port"]), extra)
+	} else {
+		hook = graylog.NewAsyncGraylogHook(fmt.Sprintf("%s:%s", h.Settings["host"], h.Settings["port"]), extra)
```
Comment (rafaeljesus) [inline hooks.go]: shouldn't this be?
```go
if async {
  graylog.NewAsyncGraylogHook
} else {
  graylog.NewGraylogHook
}
```

File: hooks.go (hunk 2 above)
Comment (vgarvardt) [reply hooks.go:17]: It should be either `tcp` or `udp`, any other value will give you an error in `net.Dial()`, so even if it is not set - empty value is fine, error will be thrown later.

File: hooks.go (hunk 3 above)
Comment (vgarvardt) [reply hooks.go]: Yup, you're right =)

**Review by rafaeljesus — APPROVED** [review]

**Comment by rafaeljesus** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/hellofresh/logging-go/pull-request/2/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/logging-go) [Approved with PullApprove](https://pullapprove.com/hellofresh/logging-go/pull-request/2/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/logging-go)

**Comment by zeelax** [comment]
+1

[Approved with PullApprove](https://pullapprove.com/hellofresh/logging-go/pull-request/2/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/logging-go) [Approved with PullApprove](https://pullapprove.com/hellofresh/logging-go/pull-request/2/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/logging-go) [Approved with PullApprove](https://pullapprove.com/hellofresh/logging-go/pull-request/2/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/logging-go)

**Pull request merged**